### DISCUSSION TITLE
why dose the model predict the same label ALL?

### DISCUSSION THREAD

**Discussion by XieXiao1997** [Project-MONAI/MONAI#3282] [Q&A]
When I try to train a 3D binary classfication refer to 3d_classification\torch\densenet_training_dict, I found that there are always only a few of the same values after a few epoch. So i cheak the ConfuseMatrix ,the predict of the model become odd like this:
Training Set:
![image](https://user-images.githubusercontent.com/61397387/140631086-82a0c056-75d1-4e96-bf9e-291bece1fbe2.png)
Validation Set:
![image](https://user-images.githubusercontent.com/61397387/140631091-0be2880c-82d9-47cc-a24f-f2d6b7c2971d.png)
It likes the training make the model using a simple way to classfy only predict the same value.
My trainset/validset‘s labels are evenly distributed. It's hard to understand.
I have tried to reduce the learning rate, add weight-decay in Adam,change the model to Resnet,but all work do not work.

**Comment by Nic-Ma** [comment]
Hi @demodoom1 ,

Thanks for your experiments here.
I think this example is only to show the functionalities of MONAI components with very few data.
Maybe you can adapt to some real-world dataset for your own requirement.

Thanks.
